Release Notes/Release Notes v2.9.0.md

```diff
@@ -1,3 +1,36 @@
 # .NET Driver Version 2.9.0 Release Notes
 
-This file is a placeholder.
\ No newline at end of file
+The main new features in 2.9.0 are:
+
+* Distributed transactions on sharded clusters
+* The sessions API supports the `IClientSession.WithTransaction()` method to conveniently run a transaction with automatic retries and at-most-once semantics.
+* Support for message compression
+* SRV polling for `mongodb+srv` connection scheme:  DNS SRV records are periodically polled in order to update the mongos proxy list without having to change client configuration or even restart the client application. This feature is particularly useful when used with a sharded cluster on MongoDB Atlas, which dynamically updates SRV records whenever you resize your Atlas sharded cluster.
+* Retryable reads: The diver can automatically retry any read operation that has not yet received any results (due to a transient network error, a "not master" error after a replica set failover, etc.). This feature feature is enabled by default.
```

Comment:
"feature feature"?  You can remove one of those

Reply:
Fixed!